ur_robot_driver/doc/features.md

```diff
@@ -1,32 +1,26 @@
-| velocity-based control                                | yes              | yes                   |
-| reporting of tcp wrench                               | yes              | yes                   |
-| reporting of tcp wrench in tcp link                   | -                | yes                   |
-| pausing of programs                                   | -                | yes                   |
-| continue trajectories after EM-Stop resume            | -                | yes                   |
-| continue trajectories after protective stop           | -                | yes                   |
-| panel interaction in between possible                 | no<sup>1</sup>   | yes                   |
-| get and set IO states                                 | yes              | yes               |
-| use tool communication on e-series                    | -                | yes                   |
-| use the driver without a teach pendant necessary      | -                | yes               |
-| support of CB2 robots                                 | yes              | -                     |
-| trajectory extrapolation on robot on missing packages | no<sup>2</sup>   | yes                   |
-| use ROS as drop-in for TP-programs                    | -                | yes   |
-| extract calibration from robot                        | -                | yes                   |
-| send custom script commands to robot                  | yes              | yes               |
-| ROS 2 support                                         | ?                | (planned)<sup>3</sup> |
-| Reconnect on a disconnected robot                     | yes              | yes           |
+| Feature                                               | ROS2 Driver
+| ---                                                   | ---                       |
+| position-based control                                | yes                       |
+| scaled position-based control                         | yes                       |
+| velocity-based control                                | no<sup>1</sup>            |
+| reporting of tcp wrench                               | not yet implemented       |
+| reporting of tcp wrench in tcp link                   | not yet implemented       |
+| pausing of programs                                   | yes                       |
+| continue trajectories after EM-Stop resume            | yes                       |
+| continue trajectories after protective stop           | yes                       |
+| panel interaction in between possible                 | yes                       |
+| get and set IO states                                 | yes                       |
+| use [tool communication forwarder](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap) on e-series | should work, not yet tested with ROS2            |
+| use the driver without a teach pendant necessary      | yes                       |
```

Comment:
Has this been verified?

Reply:
Not yet, I don't have any necessary hardware at home. Will try to organize some some in the near future. Also we should provide a ROS2 Version of the [convenience script](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/scripts/tool_communication) for easier use.